code/modules/cm_marines/equipment/guncases.dm

```diff
@@ -300,7 +300,7 @@
 	name = "\improper M707 anti-materiel rifle case"
 	desc = "A gun case containing the M707 \"Vulture\" anti-materiel rifle and its requisite spotting tools."
 	icon_state = "guncase_blue"
-	storage_slots = 7
+	storage_slots = 5
```

Comment:
Should this still be 7 and the skillless at 5?

Reply:
I really managed to do this backwards
```suggestion
	storage_slots = 7
```